### DISCUSSION TITLE
symbols-outline got archived

### DISCUSSION THREAD

**Discussion by nikolovlazar** [LazyVim/LazyVim#2304] [General]
I noticed that [symbols-outline](https://github.com/simrat39/symbols-outline.nvim) got archived yesterday due to the maintainer not having time to work on it anymore. What should we do about this?

**Comment by dpetka2001** [comment]
If you install from `:LazyExtras` the `aerial` extra, you can press `<leader>cs` to show you a symbols outline on the right similar to `symbols-outline`. You might want to try that out.

**Reply by nikolovlazar** [reply]
I'll give that a go. Thanks @dpetka2001. We should also make sure the docs reflect the archived state.

**Reply by dpetka2001** [reply]
It probably will get removed once the maintainer comes back I suppose.
